.travis.yml

```diff
@@ -41,7 +41,7 @@ before_install:
   - echo "https://$GH_TOKEN:@github.com" > .git/credentials
 
   # Manually install elasticsearch until https://github.com/travis-ci/apt-source-whitelist/issues/190
```

Comment:
I'll change that comment, since we've a different reason for using curl (to get v5)